### PR TITLE
Build a default user agent to help track down misconfigured clients

### PR DESCRIPTION
**Pull request by abloom**

### CODE REVIEW

File: src/helpers.coffee [hunk 1]
```diff
@@ -85,6 +85,12 @@ Url = require 'url'
 @buildUserAgent = (options) ->
   {serviceVersion, serviceName, appSha, appName, fqdn} = options
 
+  serviceVersion ?= "serviceVersion"
+  serviceName ?= "serviceName"
```
Comment (jkrems) [inline src/helpers.coffee]: Could this be something like `[no serviceName]` etc.. - right now the user agent looks like the product of a bug in gofer.

**Comment by jkrems** [comment]
Nice! One nitpick about the formatting, but looks good in general.

**Comment by abloom** [comment]
@jkrems updated.

**Comment by jkrems** [comment]
:shipit:

**Comment by jkrems** [comment]
v2.2.1

**Pull request merged**